### PR TITLE
PICARD-1024: Allow specifying a configuration file path

### PR DESCRIPTION
**Pull request by mineo**
This is mostly useful for debugging to avoid touching the main configuration
file while trying different settings, but could also be used for switching
between multiple tagging profiles.

https://tickets.metabrainz.org/browse/PICARD-1024

### CODE REVIEW

File: picard/ui/options/renaming.py [hunk 1]
```diff
@@ -22,11 +22,11 @@
 import sys
 from functools import partial
 from PyQt4 import QtGui
-from PyQt4.QtGui import QPalette, QColor
+from PyQt4.QtGui import QPalette
 from picard import config
 from picard.const import PICARD_URLS
 from picard.file import File
-from picard.script import ScriptParser, SyntaxError, ScriptError, UnknownFunction
+from picard.script import ScriptParser, SyntaxError, ScriptError
```
Comment (zas) [inline picard/ui/options/renaming.py]: @mineo: this cleanup is totally unrelated to the rest of the PR imho. Can you make a standalone import cleanup PR ? I agree with those trivial changes of course, but it is better to not mix them with feature PRs.

**Comment by zas** [comment]
:+1:

File: picard/ui/options/renaming.py (hunk 1 above)
Comment (mineo) [inline picard/ui/options/renaming.py]: I won't make an import cleanup PR across all of Picard because this requires a decent amount of testing because not all imports are side effect free (that is, simply removing the otherwise unused import might already run code).
